docs/writing/gc_content.py

```diff
@@ -0,0 +1,20 @@
+#!/usr/bin/env python2
```

Comment:
By default, no. At least only ``python`` is on the ``$PATH`` on macOS Sierra 10.12.5, not ``python2``.

```
$ uname
Darwin
$ which python2
$ python2
-bash: python2: command not found
$ which python
/usr/bin/python
$ python --version
Python 2.7.10
```